webapp/client/src/components/FormFieldRadioGroup.js

```diff
@@ -110,7 +110,7 @@ function FormFieldRadioGroup({
                   htmlFor={fieldId + option.value}
                   key={`${fieldId}-label-${option.value}`}
                 >
-                  {option.displayName}
+                  <span>{option.displayName}</span>
```

Comment:
```
 input[type="radio"] + label {
    white-space: pre-wrap;
    display: flex;
    padding-left: 0;
    padding-top: 0;
    font-size: var(--text-medium);
  }
```
The white-space: pre-wrap should do the trick without a span :grin:

Reply:
It doesn't. That leads to multiple problems, in fact^^
That was my first solution, too as I also used this in another PR, I think. But: This in fact makes the label *not* break. Ever. So also not if the screen is too small for it. Using a span is the solution for both problems: displaying the whitespaces (which your solution would also do) and wrapping correctly.